### PR TITLE
Rename names around Fireworks to match FireworkRocket

### LINKED ISSUE #2555: FireworkRocketEntity does not match renderer name FireworkEntityRenderer
Imo just because it's shorter FireworkRocketEntity should be changed to FireworkEntity

### PR DESCRIPTION
**Pull request by frqnny**
I could go the other way and rename everything Firework to FireworkRocket, but it depends on what you guys think. 

Closes #2555

### CODE REVIEW

**Comment by liach** [comment]
No, the entity type is called `minecraft:firework_rocket` in `EntityType` class. Do it the other way around, as yarn code is registering a `FireworkEntityRenderer` to the `EntityType.FIREWORK_ROCKET`

**Comment by frqnny** [comment]
All right, updated it and added the item to the refactor.

**Review by liach — APPROVED** [review]
Would you mind renaming `FireworkChargeItem` to `FireworkStarItem` to match the registry name like for the rockets?

**Comment by frqnny** [comment]
Sorry read that wrong, sure

**Comment by frqnny** [comment]
image
yeah i see it i'll fix it

**Review by Juuxel — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

File: mappings/net/minecraft/item/FireworkStarItem.mapping [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_1780 net/minecraft/item/FireworkChargeItem
+CLASS net/minecraft/class_1780 net/minecraft/item/FireworkStarItem
```
Comment (haykam821) [inline mappings/net/minecraft/item/FireworkStarItem.mapping:1]: Nice catch. It looks like this was changed in 1.13 as seen in `ItemInstanceTheFlatteningFix`.

**Review by haykam821 — APPROVED** [review]

**Pull request merged**